grpc-server/tinkerbell.go

```diff
@@ -68,7 +68,7 @@ func (s *server) GetWorkflowContextList(ctx context.Context, req *pb.WorkflowCon
 			WorkflowContexts: wfContexts,
 		}, nil
 	}
-	return nil, nil
+	return &pb.WorkflowContextList{WorkflowContexts: []*pb.WorkflowContext{}}, nil
```

Comment:
Can we avoid this just by removing the `nil` check on line 58? As long as `wfs` is a slice type, the empty `range` loop should have the same result.

Reply:
(remove line 58, 70, 71 .. done?)

Reply:
Did that, thanks for the suggestion.